src/System Application/App/File System/src/Account/FileAccountWizard.Page.al

```diff
@@ -0,0 +1,493 @@
+            Telemetry.LogMessage('0000CTH', StrSubstNo(CTHLbl, Rec.Connector), Verbosity::Normal, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, CustomDimensions);
+            NextStep(false);
+        end else begin
+            Telemetry.LogMessage('0000CTI', StrSubstNo(CTILbl, Rec.Connector, GetLastErrorCallStack()), Verbosity::Normal, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, CustomDimensions);
+            NextStep(true);
+        end;
+
+        if not ConnectorSucceeded then
+            Error(GetLastErrorText());
+    end;
+
+    [TryFunction]
+    local procedure TryRegisterAccount(var AccountWasRegistered: Boolean)
+    var
+        FileAccountImpl: Codeunit "File Account Impl.";
+        FileConnector: Interface "File System Connector";
+    begin
+        // Check to validate that the connector is still installed
+        // The connector could have been uninstalled by another user/session
+        if not FileAccountImpl.IsValidConnector(Rec.Connector) then
+            Error(FileConnectorHasBeenUninstalledMsg);
+
+        FileConnector := Rec.Connector;
+
+        ClearLastError();
+        AccountWasRegistered := FileConnector.RegisterAccount(RegisteredAccount);
+        RegisteredAccount.Connector := Rec.Connector;
+    end;
+
+    local procedure ShowDoneStep()
+    begin
+        DoneVisible := true;
+        BackActionVisible := false;
+        NextActionVisible := false;
+        CancelActionVisible := false;
+        FinishActionVisible := true;
+    end;
+
+    local procedure SetDefaultControls()
+    begin
+        // Actions
+        BackActionVisible := true;
+        BackActionEnabled := true;
+        NextActionVisible := true;
+        NextActionEnabled := true;
+        CancelActionVisible := true;
+        FinishActionVisible := false;
+
+        // Groups
+        WelcomeVisible := false;
+        ChooseConnectorVisible := false;
+        DoneVisible := false;
+    end;
+
+    local procedure LoadTopBanners()
+    var
+        AssistedSetupLogoTok: Label 'ASSISTEDSETUP-NOTEXT-400PX.PNG', Locked = true;
+    begin
+        if MediaResourcesStandard.Get(AssistedSetupLogoTok) and
+            MediaResourcesDone.Get(AssistedSetupLogoTok) and (CurrentClientType() = ClientType::Web)
+        then
+            TopBannerVisible := MediaResourcesDone."Media Reference".HasValue();
+    end;
+
+    var
+        RegisteredAccount: Record "File Account";
+        MediaResourcesStandard: Record "Media Resources";
+        MediaResourcesDone: Record "Media Resources";
+        [RunOnClient]
+        AppSource: DotNet AppSource;
+        Step: Option Welcome,"Choose Connector","Register Account",Done;
+        AppSourceTok: Label 'AppSource';
+        ExtensionManagementTok: Label 'Extension Management';
+        FileCategoryLbl: Label 'File', Locked = true;
+        LearnMoreURLTxt: Label 'https://go.microsoft.com/fwlink/?linkid=2134520', Locked = true;  //FIXME
```

Comment:
Fixme

Reply:
@JesperSchulz needs to be replaced with the later documentation. How should I proceed here?

Reply:
I have changed it.